content/02.intro.md

```diff
@@ -1,10 +1,12 @@
 ## Introduction {#intro}
 
 Deep learning (DL), a subfield of machine learning (ML) implementing artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
-Despite its growing popularity, DL itself remains an active area of research. Its everchanging complexity and lack of current beginner resources focused on biological applications pose large barriers of entry to newcomers who wish to utilize state-of-the-art DL in their research.
-Biological insight garnered from DL has been well-documented in the scientific literature, with applications ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z]. However, few resources articulate DL best practices to the scientific community. Most instructional literature focuses on ML broadly, rather than DL specifically, further limiting accessibility and reproducibility [@doi:10.1186/s13040-017-0155-3].
+There are many well-documented cases in the literature of novel biological insight being revealed by careful application of DL methods ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z]. 
+However, a lack of relevant up-to-date resources focused on biological applications of DL pose a significant barrier to newcomers wishing to utilize state-of-the-art DL in their research.
+As DL is an active and specialised research area, resources are rapidly rendered obsolete and few resources articulate DL best practices to the scientific community.
```

Comment:
I think we used AE elsewhere, so I would suggest changing "specialised" to "specialized"

Reply:
Going to have to set aspell to american english or I'll keep making that mistake!

Fixed.